src/kvstore/Common.h

```diff
@@ -106,26 +108,37 @@ inline std::ostream& operator<<(std::ostream& os, const Peer& peer) {
 struct Peers {
  private:
   std::map<HostAddr, Peer> peers;
+  int createdTime;
 
  public:
-  Peers() {}
+  Peers() {
+    createdTime = static_cast<int>(time::WallClock::fastNowInSec());
```

Comment:
`static_cast<int> ` can be removed? 
Use `int64_t createdTime` to avoid conversion

Reply:
int64_t is a little difficult to encode and decode?

Reply:
done